src/components/base/accordion/Accordion.js

```diff
@@ -112,45 +112,54 @@ export function Item({
         aria-controls={`accordion-unique-${eventKey}-${title}`}
         aria-expanded={isCurrentEventKey}
       >
-        <div className={styles.header_content}>
-          <div
-            className={[
-              animations["f-translate-right"],
-              // if additional text is to be shown we need to set a wwidth (.firstelement)
-              // of first element in accordion header
-              additionalTxt && styles.firstelement,
-            ].join(" ")}
-          >
-            <Link tag="span" className={styles.link_on_year} tabIndex={-1}>
-              <Title type="text2" skeleton={isLoading} lines={1} tag="h3">
-                {title}
-              </Title>
-            </Link>
-            {subTitle && (
-              <Text tag={"span"} type="text4">
-                {subTitle}
-              </Text>
-            )}
-          </div>
-          {additionalTxt && (
-            <div className={styles.textbox}>
-              {additionalTxt?.map((txt, index) => (
-                <Text tag={"span"} type="text2" key={`addition-${index}`}>
-                  {txt}
+        {CustomHeaderCompnent ? (
+          <CustomHeaderCompnent
+            onClick={onClick}
+            onKeyDown={handleKeypress}
+            expanded={isCurrentEventKey}
```

Comment:
Har gjort så man kan give Accordion en custom component som kan bruges i stedet for default, der kun tager imod tekst.

Reply:
:star: